Headers/AppKit/NSKeyValueBinding.h

```diff
@@ -86,10 +86,16 @@
 
 @end
 
-// Keys in options dictionary
+// typedefs
+typedef NSString* NSBindingName;
```

Comment:
Now you define this type twice, it is already on line 95. BTW, the placement of the star is correct on line 95 and incorrect here.